source/Calamari.Aws/Deployment/Conventions/DeleteCloudFormationStackConvention.cs

```diff
@@ -52,6 +52,8 @@ private async Task InstallAsync(RunningDeployment deployment)
         {
             Guard.NotNull(deployment, "deployment can not be null");
 
+            var deploymentStartTime = DateTime.Now;
```

Comment:
This api does not use UTC?

Reply:
The filtering is done locally after obtaining the stack events from CloudFormation, and the timestamps that come back are in our local time. Do you think it would be good to do these comparisons using UTC?

Reply:
I didn't realise the filter is dome client-side, why is that?

Reply:
Unfortunately the CloudFormation API for getting stack events only has parameters for stack name and a NextToken which appears to be used for paging, it doesn't allow any form of filtering on properties. I should probably check on the requirement around that NextToken to ensure we aren't forced to use it in this context if there are lots of events as we want to get all the relevant stack events for a run whereas previously we were only getting the latest, I'll check on this.

Reply:
Hmmm yeah looks like if the response size is > 1Mb it will send a NextToken and we'll need to request more events, will need to add support for that

Reply:
So there is no way to filter these events server side?

Reply:
No, the CloudFormation API for getting stack events only allows StackName and NextToken as parameters to the request unfortunately